src/main/java/com/julienviet/pgclient/Numeric.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (C) 2017 Julien Viet
```

Comment:
Just seeing the stale date. FWIW, we got rid of the date in [headers in Hibernate](https://github.com/hibernate/hibernate-validator/blob/4f0c69d3c6edf1e3c99a1260237e69e1bbfc9ad1/engine/src/main/java/org/hibernate/validator/internal/engine/ValidationContext.java#L1-L6) projects, as its just a hazzle to maintain.